tests/ttnn/integration_tests/squeezebert/test_performance.py

```diff
@@ -0,0 +1,155 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import time
+
+import pytest
+
+from loguru import logger
+import torch
+import transformers
+
+
+import ttnn
+from ttnn.model_preprocessing import preprocess_model_parameters
```

Comment:
`tests/ttnn/integration_tests` is a poor place for a performance test. there are some that are already in here.

That is not recommended. Please move it to a different folder within `models/experiemental/...`.

Reply:
Any action on this?

Reply:
test_performance is moved to `models/experimental/functional_squeezebert/tests/`. 
Commit link - [link](https://github.com/tenstorrent/tt-metal/compare/main...keerthana/functional_squeezebert_).
Thank you.